Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem "middleman", "~> 4"#, github: "middleman/middleman", branch: "v3-stable"
-gem 'matomo-middleman'
+gem "matomo-middleman", github: "razzeee/middleman-matomo"
```

Comment:
Best would be, if this pointed to a trusted repo. So a flatpak fork or one done by @barthalion